# request params
sensor_msgs/Image depth_image
sensor_msgs/CameraInfo camera_info
---
# response params
GraspProposals grasp_proposals
